examples/ant-design/app/entry.server.tsx

```diff
@@ -0,0 +1,23 @@
+import { renderToString } from "react-dom/server";
+import { RemixServer } from "remix";
+import type { EntryContext } from "remix";
+//imported the ant design styles
+import 'antd/dist/antd.css'
+import './styles/index.css'
```

Comment:
Hi, I saw that you've imported AntD css files two times both in the 
`client.server.tsx` and `root.tsx` 
I was curious if that was on purpose or did you forgot it?

Reply:
sorry i have done it by mistake